seismic/traveltime/requirements_conda_py37.txt

```diff
@@ -0,0 +1,28 @@
+#(hiperseispy37) @vdi-n23
+# conda install --file requirements_conda_py37.txt
+
+sphinx>=1.8
+numpy==1.16.4
+pytest>=4.2.0
+pandas
+pyproj
+h5py
+pyasdf
+obspy==1.1.0
```

Comment:
Comment to explain why it is hard pinned? Not sure what is best practice with this. I recently read that hard pinning can make it hard to solve the conda environment for a good configuration.